src/main/java/org/apache/sling/discovery/impl/standalone/NoClusterDiscoveryService.java

```diff
@@ -118,10 +127,17 @@ private void createNewView(final Type eventType, boolean inform) {
             final InstanceDescription myInstanceDescription = new InstanceDescriptionImpl(this.settingsService.getSlingId(),
                     this.cachedProperties);
             this.currentTopologyView = new TopologyViewImpl(myInstanceDescription);
+            
+            // If always offline is enabled, mark the topology as not current
+            if (this.alwaysOffline) {
+                this.currentTopologyView.setNotCurrent();
```

Comment:
is there a particular reason for introducing `setNotCurrent` or would reusing `invalidate` be an option too?